backend/copr_backend/storage.py

```diff
@@ -58,10 +59,17 @@ def init_project(self, dirname, chroot):
         """
         raise NotImplementedError
 
-    def upload_build_results(self, chroot, results_dir, target_dir_name):
+    def upload_rpm(self, repository, path):
+        """
+        Add an RPM to the storage
+        """
+        raise NotImplementedError
+
+    def upload_build_results(self, chroot, results_dir, target_dir_name, max_workers=1):
         """
         Add results for a new build to the storage
         """
+        raise NotImplementedError
```

Comment:
I don't think we can afford this exception here;  it was meant to be a no-op for the other storage.